packages/data/src/registry.js

```diff
@@ -150,7 +150,10 @@ export function createRegistry( storeConfigs = {} ) {
 
 		const namespace = createNamespace( reducerKey, options, registry );
 		registerGenericStore( reducerKey, namespace );
```

Comment:
As an aside, can we switch the variable name from `reducerKey` to `storeKey`?  It could help with #14150.  I think this pull would be an appropriate place to do it because of the refactoring taking place.

Reply:
I think it should be "storeName" but given how widespread this change could be, I think it's better in its own PR.